### PR TITLE
[Console] Add Aborted const for main exit codes

### PR DESCRIPTION
**Pull request by PululuK**
| Q             | A
| ------------- | ---
| Branch?       | 5.x 
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no 
| Tickets       | N/A
| License       | MIT
| Doc PR        | N/A

**Usage example :** 

```php

if (self::ACTION_CANCEL === $selectedActionId) {
      $output->writeln('<info>Action canceled</info>');
      return Command::ABORTED;
 }
```

### CODE REVIEW

**Comment by chalasr** [comment]
Can you please describe your use case for this?

File: src/Symfony/Component/Console/Command/Command.php [hunk 1]
```diff
@@ -32,6 +32,7 @@ class Command
     public const SUCCESS = 0;
     public const FAILURE = 1;
     public const INVALID = 2;
+    public const ABORTED = 3;
```
Comment (chalasr) [inline src/Symfony/Component/Console/Command/Command.php:35]: Should it be `130`? https://tldp.org/LDP/abs/html/exitcodes.html

Comment (PululuK) [reply src/Symfony/Component/Console/Command/Command.php:35]: Hi @chalasr 

I just add a use case in my PR description.
Here aborted means that the user has decided himself to cancel the command.

**Comment by fabpot** [comment]
I'm :-1: on this addition in core.

File: src/Symfony/Component/Console/Command/Command.php (hunk 1 above)
Comment (PululuK) [reply src/Symfony/Component/Console/Command/Command.php:35]: @fabpot  maybe we should call this `CANCELED` instead of `ABORTED` ?

**Comment by PululuK** [comment]
> I'm  on this addition in core.

Why then ?

**Comment by fabpot** [comment]
I don't want to go down this path. We've mainly added the 2 existing constants to make code more explicit (as 0 for success might be confusing for some developers). But why adding this one and not 127 for instance? Nothing forbids anyway to use it in your own code.

**Pull request closed without merging**